classic/src/view/panel/layer/General.js

```diff
@@ -54,6 +54,28 @@ Ext.define('MoMo.admin.view.panel.layer.General',{
                 bind: {
                     hidden: '{!isNewLayer}'
                 },
+                msgTarget: 'under',
+                validator: function(val) {
+                    var startsWith = val.startsWith(' ');
```

Comment:
The validator function has been moved in a new util class `MoMo.admin.util.TextfieldValidator`. Here `Ext.String.startsWith()` ist used. See 96e4936